scripts/generate-test-services.ts

```diff
@@ -132,6 +134,11 @@ async function generateTestServicesWithLocalCoreModules(
 }
 
 async function generateAll(): Promise<void> {
+  process.on('unhandledRejection', (reason, promise) => {
```

Comment:
I assume this was needed, but I am wondering why the catch in line 182 did not handle it.

Reply:
Discussed and added a comment.